proto/imuachain/oracle/v1/nonce.proto

```
@@ -1,9 +1,9 @@
 syntax = "proto3";
-package exocore.oracle.v1;
+package imuachain.oracle.v1;
 
 import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)